helm/hocs-casework/templates/_envs.tpl

```diff
@@ -0,0 +1,91 @@
+{{- define "deployment.envs" }}
+- name: JAVA_OPTS
```

Comment:
Use [YAML folding syntax](https://yaml-multiline.info/) to split this over many lines instead of one big long one.